homeassistant/components/zwave_js/discovery.py

```diff
@@ -466,3 +464,18 @@ def check_value(value: ZwaveValue, schema: ZWaveValueDiscoverySchema) -> bool:
     if schema.type is not None and value.metadata.type not in schema.type:
         return False
     return True
+
+
+@callback
+def check_device_class(
+    device_class: DeviceClassItem, required_value: Optional[Set[Union[str, int]]]
```

Comment:
Why does the required value need to be optional? We always pass it in where we use this function.

Reply:
Yeah, but it can be None